tests/tests_bond_deprecated_initscripts.yml

```diff
@@ -10,5 +10,6 @@
         network_provider: initscripts
       tags:
         - always
+        - tests::expfail
```

Comment:
Since this is a generated file, it should break the custom test: https://github.com/linux-system-roles/network/blob/main/.travis/custom.sh but it does not seem to be running according to https://github.com/linux-system-roles/network/pull/392/checks?check_run_id=2580561690#step:5:27

Reply:
> Since this is a generated file, it should break the custom test: https://github.com/linux-system-roles/network/blob/main/.travis/custom.sh but it does not seem to be running according to https://github.com/linux-system-roles/network/pull/392/checks?check_run_id=2580561690#step:5:27

Not sure why it isn't listed at https://github.com/linux-system-roles/network/blob/main/.github/workflows/tox.yml#L40-L43